modules/arch-service-catalog.adoc

```diff
@@ -0,0 +1,238 @@
+// Module included in the following assemblies:
+//
+// * architecture/service-catalog.adoc
+
+[id='service-catalog-{context}']
+= About the service catalog
+
+When developing microservices-based applications to run on cloud native
```

Comment:
@kalexand-rh I'm pulling in these first few paragraphs into a file called sb-about-service-catalog.adoc with this PR [1] in case you want to eventually rebase and adjust to include this file instead. Should be merged in the next day or so.

[1] https://github.com/openshift/openshift-docs/pull/14067

Reply:
Thanks @bergerhoffer! I think you picked a good place to split the module.